web-jehlomat/src/Components/CollapsibleText/index.tsx

```diff
@@ -0,0 +1,34 @@
+import * as S from './styles';
+import { FC, useCallback, useState } from 'react';
+import { useMediaQuery } from '@mui/material';
+import { media } from 'utils/media';
+
+interface IProps {
+    title: string;
+    text: string; // could be HTML
```

Comment:
What would you say to define a `type HtmlText = string;`?

Reply:
good